sys/tun.txt

```diff
@@ -0,0 +1,116 @@
+# Copyright 2015 syzkaller project authors. All rights reserved.
+# Use of this source code is governed by Apache 2 LICENSE that can be found in the LICENSE file.
+
+include <linux/if_tun.h>
+
+syz_open_dev$tun(dev strconst["/dev/net/tun"], id const[0], flags flags[open_flags]) fd[tun]

```

Comment:
tun file descriptor also seems to support write of several structs like virtio_net_hdr and tun_pi.
please also specialize write$tun